src/fswAlgorithms/effectorInterfaces/prescribedTrans/prescribedTrans.rst

```diff
@@ -1,32 +1,39 @@
-frame for the prescribed body is designated by the frame :math:`\mathcal{F}`. Accordingly, the prescribed states for the
-secondary body are written with respect to the mount frame, :math:`\mathcal{M}`. The prescribed states are: ``r_FM_M``,
-``rPrime_FM_M``, ``rPrimePrime_FM_M``, ``omega_FM_F``, ``omegaPrime_FM_F``, and ``sigma_FM``. Because this is a
-purely translational profiler, the states ``omega_FM_F``, ``omegaPrime_FM_F``, and ``sigma_FM`` are held
-constant in this module.
-
-To use this module for prescribed motion purposes, it must be connected to the :ref:`PrescribedMotionStateEffector`
-dynamics module in order to profile the states of the secondary body. The required maneuver is determined from the
-user-specified scalar maximum acceleration :math:`a_{\text{max}}`, the mount frame axis for the translational motion,
-the prescribed body's initial position vector with respect to the mount frame :math:`\boldsymbol{r}_{F/M}(t_0)`, and
-the reference position vector or the prescribed body with respect to the mount frame
-:math:`\boldsymbol{r}_{F/M} (\text{ref})`.
-
-The maximum scalar acceleration is applied constant and positively for the first half of the maneuver and
-constant negatively for the second half of the maneuver. The resulting velocity of the prescribed body is
-linear, approaching a maximum magnitude halfway through the maneuver and ending with zero residual velocity.
-The corresponding translational trajectory the prescribed body moves through during the maneuver is parabolic in time.
-
+This module profiles a 1 DOF translation for a rigid body connected to a rigid spacecraft hub. The body frame
+of the secondary body is designated by the frame :math:`\mathcal{F}`. The secondary body states are profiled
+relative to a hub-fixed frame :math:`\mathcal{M}`. The :ref:`PrescribedMotionMsgPayload` message
+is used to output the prescribed states from the module. The prescribed states are: ``r_FM_M``, ``rPrime_FM_M``,
+``rPrimePrime_FM_M``, ``omega_FM_F``, ``omegaPrime_FM_F``, and ``sigma_FM``. Because this module only profiles
+a translation, the secondary body rotational states ``omega_FM_F``, ``omegaPrime_FM_F``, and ``sigma_FM`` are fixed
+in this module.
+
+.. important::
+    Note that this module assumes the initial and final secondary body translational rates are zero.
+
+The general inputs to this module that must be set by the user are the secondary body rotational states ``omega_FM_F``,
+``omegaPrime_FM_F``, and ``sigma_FM``, the secondary body translational axis expressed in Mount frame components
+``transAxis_M``, the initial scalar position of the secondary body relative to the Mount frame ``transPosInit``,
+the reference position relative to the Mount frame ``transPosRef``, the maximum translational acceleration for the
+translation ``transAccelMax``, and a boolean toggle variable ``coastOption`` for selecting which of two profiling

```

Comment:
please state in the documentation that the `coastOption` flag is optional, and by default it is turned off.